riotfile.py

```diff
@@ -201,6 +201,11 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
             pys=select_pys(),
             command="pytest {cmdargs} tests/appsec",
         ),
+        Venv(
```

Comment:
I don't think we need another entry for this, it can be grouped under `tests/appsec`

Reply:
Ack, will do